software/clipbucket.yml

```diff
@@ -0,0 +1,12 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr/en/"
+source_code_url: "https://github.com/MacWarrior/clipbucket-v5"
+description: "Start your own video sharing website (YouTube/Netflix Clone) in a matter of minutes."
+licenses:
+  - AAL
+platforms:
+  - Docker
+  - PHP
+tags:
+  - Media Streaming - Multimedia Streaming
+  - Photo and Video Galleries
```

Comment:
There is some confusion as to what these categories are for. I think ClipBucket is only relevant to `Media Streaming - Multimedia Streaming`, and `Photo and Video Galleries` should be renamed to just `Photo galleries`.

Approving anyway, will handle this later.

Reply:
https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/1074